vali_objects/position.py

```diff
@@ -2,14 +2,19 @@
 import logging
 from copy import deepcopy
 from typing import Optional, List
-from pydantic import model_validator, BaseModel, Field
+from pydantic import model_validator, BaseModel, Field, model_serializer, root_validator
 
-from time_util.time_util import TimeUtil
+from time_util.time_util import TimeUtil, MS_IN_8_HOURS
 from vali_config import TradePair
 from vali_objects.vali_dataclasses.order import Order
 from vali_objects.enums.order_type_enum import OrderType
 
 import bittensor as bt
+import math
+
+CRYPTO_CARRY_FEE_PER_INTERVAL = math.exp(math.log(1 - 0.08) / (365.0*3.0))  # 8% per year for 1x leverage. Each interval is 8 hrs
```

Comment:
can just use a static number and leave the expression as a comment in case people are curious where it comes from: 0.9999238553110819

Reply:
I prefer leaving the formula as a live value to prevent a wrong constant being used if we tweak later on.